app/boards/shields/nice_view_oled/Kconfig.shield

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+config SHIELD_NICE_VIEW_OLED
+	def_bool $(shields_list_contains,nice_view_oled)
```

Comment:
As I think about this... Still unsure about the name.

OLED...

- adapter?
- bodge
- converter

Some alternative ideas.